db/schema.rb

```diff
@@ -40,23 +40,25 @@
     t.datetime "start_time"
     t.datetime "end_time"
     t.string "video_link"
+    t.string "message"
     t.boolean "confirmed", default: false
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.string "message"
     t.bigint "class_room_id"
     t.index ["class_room_id"], name: "index_lectures_on_class_room_id"
     t.index ["professional_id"], name: "index_lectures_on_professional_id"
   end
 
   create_table "professionals", force: :cascade do |t|
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
     t.string "company"
     t.string "title"
     t.string "description"
     t.string "city"
     t.string "linkedin_url"
+    t.bigint "user_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["user_id"], name: "index_professionals_on_user_id"
   end
```

Comment:
Why did you change the schema? @grantye

Reply:
I didn't change any schema. Were they from the old versions?